src/formats/rabbitmq_log.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://lnav.org/schemas/format-v1.schema.json",
+    "rabbitmq_log": {
+        "regex": {
+            "connection": {
+                "pattern" : "^=(?<level>[A-Z]+) REPORT==== (?<timestamp>\\d\\d?-\\w{3}-\\d{4}::\\d{2}:\\d{2}:\\d{2}) ===(\\n(?<body>(closing AMQP connection <(?<stuff>.+)> \\((?<sourceIP>\\d[\\d.]+\\d?):(?<sourcePort>\\d+) -> (?<destIP>\\d[\\d.]+\\d?):(?<destPort>\\d+)\\))?(.*\\n)*))?$"
```

Comment:
I wonder if the performance issue could be mitigated by tweaking this a bit.  For example changing the `.+` inside `<(?<stuff>.+)>` to `[^>]+` so that the regexp matching engine doesn't have to do any backtracking.  Just a guess, not sure if it will help!